src/disclosures/js/views/financial-view.js

```diff
@@ -619,6 +624,19 @@ var financialView = {
   },
 
   /**
+   * Sets visibility of graduation cohort content. Hidden if not available.
+   * @param {boolean} visibility - Whether or not a graduation cohort
+   * was provided
+   */
+  graduationCohortVisible: function( visibility ) {
```

Comment:
Also, you could update `visibility` to `isVisible`, so when reading the variable name the answer is either true or false.